docs/performance/selects.txt

```diff
@@ -0,0 +1,90 @@
+==================
+Select Performance
+==================
+
+
+Aggregations and Group By
+=========================
+
+A common use case for CrateDB is to do ``GROUP BY`` queries on tables for
+analytic purposes. Say you're selecting the ``avg``, ``max`` and ``min`` of
+some measurements over a billion records and group them by a device id.
+
+By default CrateDB processes all matching records, which may require a lot of
+processing power depending on the data set and the size of the CrateDB cluster.
+
+Similar products sometimes limit the amount of records that are processed for
+GROUP BY operations. That means the user might get inaccurate results, but for
```

Comment:
I believe that "inaccurate results" sound a little too harsh?
How about "less accurate".
Or "simialar products trade accuracy for performance" ?